lib/app/listener.ex

```diff
@@ -0,0 +1,39 @@
+defmodule App.Listener do
```

Comment:
@RobStallion the `Listener` implementation looks good. 
(_but there aren't any comments in this file ..._ 🤔)